porthtml.html

```diff
@@ -78,15 +79,297 @@ <h6 class="LET"><a href="#">LET’S CONNECT</a></h6>
          
           
         </ul>
-        <img src="abc.svg" alt="" width="20" height="20">
+        </div>
+</div>
+     <br>
+     <br>
```

Comment:
- [ ] Kindly use appropriate margin and padding remove all be tags from from website so it is more semantic